### PR TITLE
Added snippet for pulse-loader

### PR DESCRIPTION
**Pull request by prashrey**
Would like to add a pulse loader made using CSS Animation property

### CODE REVIEW

File: snippets/pulse-loader.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+```html
+<div class="ripple-loader">
+    <div></div>
+    <div></div>
+</div>
+```
+
+```css
+    .ripple-loader {
```
Comment (Chalarangelo) [inline snippets/pulse-loader.md]: Please fix the indentation. Top level rules should have `0` spaces and every subsequent level should be indented by `2` spaces more than the last one.

File: snippets/pulse-loader.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: Pulse loader
+tags: animation
+---
+
+Creates a pulse effect loader animation using `animation-delay` property.
+
+```html
+<div class="ripple-loader">
+    <div></div>
```
Comment (Chalarangelo) [inline snippets/pulse-loader.md]: Please fix indentation for the inner `<div>` elements to be `2` spaces.

File: snippets/pulse-loader.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: Pulse loader
+tags: animation
```
Comment (Chalarangelo) [inline snippets/pulse-loader.md]: Please add an expertise level, like so:

```suggestion
tags: animation, intermediate
```

File: snippets/pulse-loader.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            top: 0;
+            left: 0;
+            width: 64px;
+            height: 64px;
+            opacity: 0;
+        }
+    }
+```
+
+
```
Comment (Chalarangelo) [inline snippets/pulse-loader.md]: Please trim the extra empty line.

File: snippets/pulse-loader.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+#### Explanation
```
Comment (Chalarangelo) [inline snippets/pulse-loader.md]: Explanation might need a final rephrase, but I'll take care of it when the rest of the issues are resolved.

**Comment by prashrey** [comment]
Thanks! I'll make the changes.

**Comment by prashrey** [comment]
@Chalarangelo I've checked in the changes. Please let me know if any further improvement is required. Thanks!

**Pull request merged**